sdk/app_cpu1/user/usr/user_config.h

```diff
@@ -49,4 +49,16 @@
 // set to 1 for enabled, 0 for disabled
 #define USER_CONFIG_ENABLE_AMDS_SUPPORT (0)
 
+// Scheduler Interval Tolerance Override
+// as of AMDC Firmware v1.3, all timing variables for tasks (runtime, loop time, etc)
+// are stored as double-precision floating point values, which have precision errors
+// that require a margin of tolerance. sometimes non-default PWM frequency and/or
+// timing manager ratios may necessitate using the below defines to override the
+// default tolerance in common/sys/scheduler.h
+#define USER_CONFIG_OVERRIDE_SCHEDULER_INTERVAL_TOLERANCE (0)
+
+#if USER_CONFIG_OVERRIDE_SCHEDULER_INTERVAL_TOLERANCE == 1
+#define USER_CONFIG_SCHEDULER_INTERVAL_TOLERANCE_USEC (-0.15)
+#endif
```

Comment:
Hmm, I think this is a little complicated on the defines---two defines, one true/false as 1/0, one with real values.

I would encourage using `#ifdef` statements and just having the user uncomment the `USER_CONFIG_SCHEDULER_INTERVAL_TOLERANCE_USEC` value if they want to override it